views/workqueue/index.html.twig

```diff
@@ -60,9 +60,11 @@
                 <th rowspan="2" class="col-group-default">Participant Status</th>
                 <th rowspan="2" class="{{  app.isDvType ? 'col-group-default' : 'col-group-info' }}">Participant Origination</th>
                 <th rowspan="2" class="col-group-default col-group-info">Consent Cohort</th>
+                <th rowspan="2" class="col-group-default col-group-info">First Primary Consent</th>
                 <th rowspan="2" class="col-group-default col-group-info">Primary Consent</th>
                 <th rowspan="2" class="col-group-default col-group-info">Program Update</th>
                 <th rowspan="2" class="col-group-default col-group-info">Language of Primary Consent</th>
+                <th rowspan="2" class="col-group-default col-group-info">First EHR Consent</th>
```

Comment:
The JIRA ticket says to add these columns to the "show all" group, which I assume means not assigned to the default or info view groups.